src/client/src/css/jobListing.css

```diff
@@ -253,53 +190,173 @@
   font-size: large;
 }
 
-@media all and (max-width: 690px) {
-  .jobPosts_Container {
+@media all and (max-width: 905px) {
+  .jobPosts {
+    border: 2px #19718d solid;
+    width: 85%;
+    height: 570px;
+    border-radius: 15px;
+    margin: auto;
+    margin-top: 30px;
+    display: inline-block;
+  }
```

Comment:
I'm not sure if there's supposed to be a bottom margin with the screen width less than 905 pixels as well
![image](https://user-images.githubusercontent.com/78051064/230804232-12273a00-9ec8-4109-be04-4c6a7fa604b5.png)

Reply:
Fixed